railgun/internal/diagnostics/server.go

```diff
@@ -0,0 +1,55 @@
+package diagnostics
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/kong/kubernetes-ingress-controller/railgun/internal/mgrutils"
+	"github.com/sirupsen/logrus"
+)
+
+const diagnosticsServerPort = 10254
```

Comment:
You'll want to move this constant to `railgun/pkg/config/consts.go` and change it to 10256 to not conflict with the other. 1.x served everything on the same port, but we can't do that while using the Kubebuilder default health and metrics system.

Reply:
addressed